ltiauthenticator/lti11/auth.py

```diff
@@ -78,6 +92,3 @@ def authenticate(self, handler, data) -> dict:
                     k: v for k, v in args.items() if not k.startswith("oauth_")
                 },
             }
-
-    def login_url(self, base_url):
```

Comment:
Is this not needed anymore?

Reply:
@yuvipanda yes it is, I just [changed the location](https://github.com/jgwerner/ltiauthenticator/blob/refactor-validator/ltiauthenticator/lti11/auth.py#L41) so that it's above the overridden `authenticate` function.